content/300-guides/200-deployment/301-edge/450-deploy-to-cloudflare-workers.mdx

```diff
@@ -242,10 +242,20 @@ You are now ready to generate a Prisma Client.
 
 Next, you'll generate a Prisma Client that connects through the [Data Proxy](/data-platform/classic-projects/data-proxy) over HTTP.
 
+```terminal
+npx prisma generate --no-engine
```

Comment:
Is this even necessary or will Cloudflare just ignore the engine file on deploy?

Reply:
My understanding/ assumption is that the `--data-proxy` flag does/did not generate the engine file. Therefore, there was no engine file that is bundled during the deployment step.

Reply:
Confirmed, the `--data-proxy` flag does not generate any engine files, meaning there is no engine file included in the deployed application

Reply:
Not what I asked about.

Does Cloudflare maybe not copy over a file if it was there anyway because it can figure out it is not needed?

Reply:
CF bundling isn't smart. You could manually exclude the file, or you could use `--no-engine`.